docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/what-about-cloud-native-applications.md

```diff
@@ -36,7 +36,10 @@ The [Twelve-Factor App](https://12factor.net/) (a collection of patterns that ar
 
 The [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) is a primary promoter of cloud-native principles. Microsoft is a [member of the CNCF](https://azure.microsoft.com/blog/announcing-cncf/).
 
-For a sample definition and for more information about the characteristics of cloud-native applications, see the Gartner article [How to architect and design cloud-native applications](https://www.gartner.com/doc/3181919/architect-design-cloudnative-applications). For specific guidance from Microsoft about how to implement a cloud-native application, see [.NET microservices: Architecture for containerized .NET applications](https://aka.ms/microservicesebook).
+For detailed guidance on how to design and develop cloud-native applications, read the following free e-Books:
+
+*  [Architecting Cloud-Native .NET Applications for Azure](https://docs.microsoft.com/dotnet/architecture/cloud-native/introduction)
+*  [.NET Microservices: Architecture for containerized .NET applications](https://aka.ms/microservicesebook).
```

Comment:
why not link to the book here as well to be consistent? Also, if the reader is reading this page in another language, you're sending them to an English PDF instead of the material in their language.

```suggestion
* [.NET Microservices: Architecture for containerized .NET applications](../../microservices/index.md).
```

Reply:
@nishanil asked at some point to use the aka link for download metrics.

Also, English is the only language that's being generated to PDF.

Will ping him directly regarding this one.

Reply:
I agree with Maira's comments. Let's be consistent in the links.